src/singletons/Settings.hpp

```diff
@@ -318,7 +318,13 @@ class Settings : public ABSettings, public ConcurrentSettings
     QStringSetting streamlinkPath = {"/external/streamlink/customPath", ""};
     QStringSetting preferredQuality = {"/external/streamlink/quality",
                                        "Choose"};
-    QStringSetting streamlinkOpts = {"/external/streamlink/options", ""};
+    QStringSetting streamlinkOpts = {
+        "/external/streamlink/options",
+        "--twitch-low-latency --twitch-disable-ads"};
```

Comment:
We should not be setting default options for an external application

Reply:
I went ahead and removed them, but the `--twitch-low-latency` option is pretty important for the experience to match the main website. I think most users would appreciate at least this recommendation, but I understand.

Reply:
Low latency looks like something that should be in the quality chooser dialog, not hidden somewhere deep in the settings of the application. IMO

Reply:
What about adding these as "checkboxes" for the options so the user can easily toggle these, and then optionally add more. The issue there is that they might break if the streamlink project renames them at any point.
![image](https://user-images.githubusercontent.com/2222562/121361584-6b775680-c903-11eb-8629-313de0a53994.png)

Reply:
I went an added some nice toggles for the user to use.
![image](https://user-images.githubusercontent.com/2222562/122632146-dad90d00-d09e-11eb-8030-fd8a8849efb0.png)